src/Display/Modal/Modal.tsx

```diff
@@ -13,144 +13,115 @@ import {
   ModalFooter,
 } from '../../Style/Display/ModalStyle';
 
-class Modal extends React.Component<Props, State> {
-  modalContentAreaRef: React.RefObject<HTMLDivElement>;
-  mouseDownTarget: HTMLElement;
-  escEvent: (e: KeyboardEvent) => any;
-
-  state = {
-    isOpen: false,
-  };
-
-  constructor(props: Props) {
-    super(props);
-    this.modalContentAreaRef = React.createRef();
-  }
-
-  registerEventEscapeListener() {
-    if (!this.props.isVisible) return;
-    this.escEvent = escEvent(this.props.onClose);
-    document.addEventListener('keydown', this.escEvent, false);
-  }
-
-  componentDidMount() {
-    this.registerEventEscapeListener();
-  }
-
-  static getDerivedStateFromProps(nextProps: Props, prevState: State) {
-    if (nextProps.isVisible && !prevState.isOpen) {
-      document.body.style.overflow = 'hidden';
```

Comment:
do we still need this `hidden` style?

Reply:
Thanks for catching this, should not have deleted it.

Reply:
Thanks for catching this, should not have deleted it!